home/static/css/admin_style.css

```diff
@@ -0,0 +1,257 @@
+* {
```

Comment:
Can we consolidate the styles, some of the style you have here, or at least similar style, have been written by other members. Just to keep it all consistent, you can refactor and reuse the styles from the existing .css, and only keep here the style specific to the admin page if any